analyzers/golang/testdata/demotags/tags/combo.go

```diff
@@ -2,8 +2,8 @@
 
 package tags
 
-import "imports/combo"
+import "fmt"
```

Comment:
Good call adding standard lib here.

Reply:
Yeah, the alternative is we could've picked non-overlapping dependencies in `go.mod`, but I didn't want that to cause more mysterious breaks down the line when we change the CLI's actual dependencies.